src/install/ibm-mq/windows/install-msi.mdx

```diff
@@ -0,0 +1,11 @@
+---
+componentType: default
+headingText: Download using MSI
+---
+
+1. Download the latest .MSI installer image for the desired integration [from our repository](https://download.newrelic.com/infrastructure_agent/windows/integrations/).
+2. In an admin account, run the install script using an absolute path.
```

Comment:
Be consistent about how you refer to running terminal commands. I.e., "Run the following command", "run this command", "run the script", etc. should all be modified to one sentence syntax